docs/guides/disk-autoscaler.md

```diff
@@ -0,0 +1,81 @@
+---
+page_title: "Scale disk storage automatically"
+---
+
+# Scale disk storage automatically
+
+[Disk autoscaler](https://aiven.io/docs/platform/howto/disk-autoscaler#disable-disk-autoscaler) automatically increases
+the storage capacity of your Aiven service when it's running out of space. Disk autoscaler only increases storage, it doesn't scale down.
+
+You can enable disk autoscaler when you create a service or on an existing service. To enable autoscaler, create
+an autoscaler integration endpoint using the `aiven_service_integration_endpoint` resource and integrate
+your service with it using the `aiven_service_integration` resource.
+
+## Enable disk autoscaler when creating a service
+
+Follow [the disk autoscaler example](https://github.com/aiven/terraform-provider-aiven/tree/main/examples/autoscaler_integration)
+to learn how to to create an Aiven for PostgreSQL® service, autoscaler endpoint, and enable the autoscaler integration on the service.
+
+## Enable disk autoscaler on an existing service
+
+1. If the service has `additional_disk_space` configured, remove the field. The additional storage will be adjusted by the autoscaler.
+
+2. Create an autoscaler endpoint using the `aiven_service_integration_endpoint` resource. In the following example, an endpoint is created with a maximum total disk storage of 200GiB:
+
+      ```hcl
+      resource "aiven_service_integration_endpoint" "autoscaler_endpoint" {
+        project       = var.aiven_project.example_project.project
+        endpoint_name = "disk-autoscaler-200GiB"
+        endpoint_type = "autoscaler"
+
+        autoscaler_user_config {
+          autoscaling {
+            cap_gb = 200
+            type   = "autoscale_disk"
+          }
+        }
+      }
+      ```
+
+3. Enable the autoscaler integration on your service using the `aiven_service_integration` resource. For example:
+
+      ```hcl
+      resource "aiven_service_integration" "autoscaler_integration" {
+        project                 = var.aiven_project.example_project.project
+        integration_type        = "autoscaler"
+        source_service_name     = "SERVICE_NAME"
+        destination_endpoint_id = aiven_service_integration_endpoint.autoscaler_endpoint.id
+      }
+      ```
+
+    Where `SERVICE_NAME` is the name of the service to scale.
+
+4. To preview your configuration changes, run:
+
+    ```bash
+    terraform plan
+    ```
+
+5. To create the endpoint and enable the autoscaler, apply the configuration by running:
+
+    ```bash
+    terraform apply --auto-approve
+     ```
+
+## Disable disk autoscaling
+
+To disable the disk autoscaler without destroying the endpoint or service:
+
+1. Remove the `aiven_service_integration` for the autoscaler. Keep the `aiven_service_integration_endpoint` for the autoscaler.
```

Comment:
Hey. Probably having an endpoint without an integration doesn't make much sense.

Reply:
My inital thought was that someone might want to use the endpoint for a different service, but this doesn't really warrant having separate instructions for anyway. 👍